docs/source/optimization/interface.rst

```diff
@@ -0,0 +1,23 @@
+minimize and maximize
+=====================
+
+.. currentmodule:: estimagic.optimization.optimize
+
+Both optimization functions have the exactly same interface. In fact, ``maximize``
+ just switches the sign of your criterion function and calls ``minimize``.
+
+
+.. autofunction:: minimize
```

Comment:
The constraints argument is not properly rendered. It just says "See for details" without a link to click on.

Reply:
Should be fixed.